run_scans.py

```diff
@@ -259,7 +277,6 @@ def launchArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax, nev
       res = pool.map_async(launch,
                            itertools.izip([options.tool for x in range(len(chamber_config))],
                                           [options.shelf for x in range(len(chamber_config))],
-                                          [options.slot for x in range(len(chamber_config))],
```

Comment:
At the start of this block, e.g. just before:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/pull/197/files#diff-5190e78d72218d0020a29344d4b44bdbR270

You should add a print statement that says:

```python
print("Cannot run links in parallel, there is only one VFAT_DAQ_MONITOR")
exit(os.EX_USAGE)
```

Reply:
Great, see this was done already.